src/components/map.js

```diff
@@ -1,49 +1,83 @@
-import React, { useRef, useEffect, useState } from "react";
```

Comment:
Consider changing this to typescript if you have time.

Reply:
Sure, going to rewrite the component shortly and recommit